### ISSUE TITLE
unsupported Ruby engine `jruby'... on JRuby

### ISSUE THREAD

**Issue by ileitch** [headius/ruby-atomic#17]
I installed atomic yesterday, I'm ensure what version it was at the time but it installed OK. I see there was a new release very recently.

I was having a weird issue where a CAS would stall, so I went to upgrade.

First weird issue is that gem install was asking me to enabled C-extensions. I never had to do that before. Anyway, I see cext.enabled. Now I'm getting:

```
➜  bm_test  ruby -e 'require "rubygems"; require "atomic"'
/Users/ianleitch/.rvm/gems/jruby-1.7.3/gems/atomic-1.1.4/lib/atomic/fallback.rb:1: unsupported Ruby engine `jruby', using less-efficient Atomic impl
```

```
➜  bm_test  ruby -v
jruby 1.7.3 (ruby-1.8.7p370) 2013-02-21 dac429b on Java HotSpot(TM) 64-Bit Server VM 1.8.0-ea-b84 +indy [darwin-x86_64]
```

**Comment by headius** [comment]
Sorry, I forgot to push the -java version of the gem. rubygems.org seems to be down at the moment, though, so I'll push it tomorrow.

**Comment by headius** [comment]
1.1.4-java has been pushed. Sorry for the inconvenience!

**Comment by ileitch** [comment]
Thank you.

**Comment by ileitch** [comment]
I'm still seeing the fallback warning with -java.

```
$ gem install atomic
Fetching: atomic-1.1.5-java.gem (100%)
Successfully installed atomic-1.1.5-java
1 gem installed
$ ruby -e "require 'rubygems';require 'atomic'"
/Users/ian/.rvm/gems/jruby-1.7.3/gems/atomic-1.1.5-java/lib/atomic/fallback.rb:1: unsupported Ruby engine `jruby', using less-efficient Atomic impl
$ ruby -v
jruby 1.7.3 (1.9.3p385) 2013-02-21 dac429b on Java HotSpot(TM) 64-Bit Server VM 1.8.0-ea-b84 +indy [darwin-x86_64]
```

**Comment by headius** [comment]
OK I'll look into it again.
- Charlie (mobile)

**Comment by headius** [comment]
Bleh, I released 1.1.5 without the java bits in it. Released 1.1.6 and confirmed it's working.
